pkg/webhook/elasticquota/validating/validating_handler.go

```diff
@@ -125,3 +126,12 @@ func (h *ElasticQuotaValidatingHandler) InjectCache(cache cache.Cache) error {
 	})
 	return nil
 }
+
+func (h *ElasticQuotaValidatingHandler) RegisterDebugApi(engine *gin.Engine) {
+	routes := engine.Group("/apis/v1")
```

Comment:
The `/apis/v1` URL prefix should be directly bound externally, and the provider only needs to consider which endpoints to support. If it's the DEBUG API, we might consider changing the name.

Reply:
fixed